### ISSUE TITLE
ExceptionClassName not defined in TypeScript Client Generation

### ISSUE THREAD

**Issue by radicalapps** [RicoSuter/NSwag#2434]
Hi,

Using NSwagStusdio 13.1.2.0 to generate a TypeScript (Fetch) client, the ExceptionClassName appears to be undefined, causing issues with the generated code. 

The generated output looks like the below, where the class name is missing.

```
export class  extends Error {
    message: string;
    status: number; 
    response: string; 
    headers: { [key: string]: any; };
    result: any; 

    constructor(message: string, status: number, response: string, headers: { [key: string]: any; }, result: any) {
        super();

        this.message = message;
        this.status = status;
        this.response = response;
        this.headers = headers;
        this.result = result;
    }

    protected is = true;

    static is(obj: any): obj is  {
        return obj.is === true;
    }
}

function throwException(message: string, status: number, response: string, headers: { [key: string]: any; }, result?: any): any {
    if (result !== null && result !== undefined)
        throw result;
    else
        throw new (message, status, response, headers, null);
}

```
This causes complication issues in the generated code. Adding in a class name where it is missing allows the code to compile without issues.

**Comment by RicoSuter** [comment]
Seems quite strange and i cannot renember a change which could lead to this regression. Are you sure the config is not empty?

**Comment by radicalapps** [comment]
Hi Rico,
I found the cause: 

I edited the NSwag file and adding in a class name, then the code generation worked as expected. 

Just now, looking back in the NSwagStudio settings, I can see the exception class name that I added in. It seems that in the latest NSwagStudio the label is missing from the Exception class name input field, so there is an input there with no label to indicate what it is. I must have seen what looked like a random input field with a class name in it, and deleted the class name. 

![image](https://user-images.githubusercontent.com/22747259/65820479-7c606100-e221-11e9-82be-e96f5a3e3e51.png)

Jeff.
